shared/chat/conversation/attachment-popup/container.js

```diff
@@ -1,13 +1,23 @@
 // @flow
+import {compose, withState, withProps} from 'recompose'
 import RenderAttachmentPopup from './'
 import {connect} from 'react-redux'
 import {navigateUp} from '../../../actions/route-tree'
 
 import type {TypedState} from '../../../constants/reducer'
 
-export default connect(
-  (state: TypedState, {routeProps}) => routeProps,
-  (dispatch: Dispatch) => ({
-    onClose: () => dispatch(navigateUp()),
-  })
+export default compose(
```

Comment:
You don't have to make this change but when I was doing my recompose work I ended up leaving the connect wrapping in the container.js and did my recompose wrapping in a wrapped export of the renderered component. It cleaned up the connect file (often complex) and had the benefit of making the dumb component function (like the state would actually work in the dumb sheet)